src/test/ui/async-await/issues/issue-67893.stderr

```diff
@@ -6,17 +6,13 @@ LL |     g(issue_67893::run())
    |
    = help: within `impl Future<Output = ()>`, the trait `Send` is not implemented for `MutexGuard<'_, ()>`
 note: future is not `Send` as this value is used across an await
-  --> $DIR/auxiliary/issue_67893.rs:9:26
+  --> /rustc/FAKE_PREFIX/src/test/ui/async-await/issues/auxiliary/issue_67893.rs:9:26
    |
-LL |     f(*x.lock().unwrap()).await;
-   |        ----------------- ^^^^^^ await occurs here, with `x.lock().unwrap()` maybe used later
-   |        |
-   |        has type `MutexGuard<'_, ()>` which is not `Send`
-note: `x.lock().unwrap()` is later dropped here
-  --> $DIR/auxiliary/issue_67893.rs:9:32
-   |
-LL |     f(*x.lock().unwrap()).await;
-   |                                ^
+   |   = note: await occurs here, with the value maybe used later
+  
+ = note: has type `MutexGuard<'_, ()>` which is not `Send`
+note: the value is later dropped here
+  --> /rustc/FAKE_PREFIX/src/test/ui/async-await/issues/auxiliary/issue_67893.rs:9:32
```

Comment:
This one is weird. It's not libstd, but just an auxiliary dependency

Reply:
`-Zsimulate-remapped-rust-src-base` seems to be affecting more than libstd paths

Reply:
Ah I figured it out, see the fixup commit (or just the full PR diff).  I will squash before landing